arviz/plots/mcseplot.py

```diff
@@ -36,9 +34,8 @@ def plot_mcse(
     extra_kwargs=None,
     text_kwargs=None,
     **kwargs
-    # fmt: on
 ):
-    """Plot quantile, local or evolution of effective sample sizes (ESS).
+    """Plot Monte Carlo Standard Error
```

Comment:
@OriolAbril Did I do this right?

Reply:
The plot is actually quantile or local MCSE, to get the mean MCSE the `mcse` function should be used. I would write `Plot quantile or local Monte Carlo Standard Error.` 

Also, thanks for noticing! This is me copy-pasting too agressively from `plot_ess`